.gitmodules

```diff
@@ -4,3 +4,11 @@
 [submodule "src/instrumentation_measurement/evmone"]
 	path = src/instrumentation_measurement/evmone
 	url = ../evmone.git
+[submodule "src/instrumentation_measurement/geth_benchmark"]
+	path = src/instrumentation_measurement/geth_benchmark
+	url = https://github.com/JacekGlen/go-ethereum
+	branch = imapp_benchmark
+[submodule "nethermind_benchmark"]
+	path = src/instrumentation_measurement/nethermind_benchmark
+	url = https://github.com/JacekGlen/nethermind
+	branch = imapp_benchmark
```

Comment:
for non-benchmark flavor of measurement, do I still use the same branch in `nethermind`?

Reply:
One branch with two projects Imapp.Benchmark.Runner and Imapp.Measurement.Runner

Reply:
cool, same as `go-ethereum`, please fork in `imapp-pl` so we can have access if need be.